website/docs/plugin/framework/migrating/providers/index.mdx

```diff
@@ -323,14 +323,18 @@ This code implements the `example_attribute` attribute for the `example_Block` b
 ```go
 func (p *exampleProvider) Schema(_ context.Context, _ provider.SchemaRequest, resp *provider.SchemaResponse) {
     resp.Schema = schema.Schema{
-        Attributes: map[string]schema.Attribute{
+    	/*...*/
+        Blocks: map[string]schema.Block{
             "example_block": schema.SingleNestedBlock{
```

Comment:
The above block is a list block and the below implementation correctly was updated to use the list validator 👍 

```suggestion
            "example_block": schema.ListNestedBlock{
```